src/librustc/ty/context.rs

```diff
@@ -1024,7 +1026,11 @@ pub struct GlobalCtxt<'tcx> {
     /// Export map produced by name resolution.
     export_map: FxHashMap<DefId, Lrc<Vec<Export>>>,
 
-    hir_map: hir_map::Map<'tcx>,
+    pub hir_forest: hir::map::Forest,
+
+    pub hir_defs: hir::map::Definitions,
```

Comment:
Random note: I think `Definitions` should be moved out of `hir::map` or even `hir`.
Long-term, `rustc::def::{DefId, Definitions, ...}` makes more sense to me.